src/librustc_mir/interpret/eval_context.rs

```diff
@@ -1406,18 +1407,17 @@ impl<'a, 'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> EvalContext<'a, 'mir, 'tcx, M
 
     /// This function checks the memory where `ptr` points to.
     /// It will error if the bits at the destination do not match the ones described by the layout.
-    ///
-    /// Note: this function does not transitively dereference other pointers. All that is checked
-    /// for pointers is whether they are not allowed to be zero, but are.
     pub fn validate_ptr_target(
```

Comment:
Well it still doesn't transitively do anything itself, it just fills `seen` and `todo` accordingly, right? The docs should say that.